front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -483,6 +486,7 @@ export default function AssistantBuilder({
     }
 
     await router.push(`/w/${owner.sId}/builder/assistants`);
+    setIsSavingOrDeleting(false);
```

Comment:
same here actually, if we crash while deleting it will stay in "loading" no ?

Reply:
yup updated